packages/server/src/arpa_reporter/services/full-file-export.js

```diff
@@ -0,0 +1,119 @@
+            a.name AS agency_name,
+            'EC' || u1.ec_code AS ec_code,
+            rp.name AS reporting_period_name,
+            CASE
+                WHEN u1.invalidated_at IS NOT NULL THEN 'Invalidated at ' || invalidated_at || ' by ' || ui.email
+                WHEN u1.validated_at IS NOT NULL THEN 'Validated at ' || validated_at || ' by ' || uv.email
+                ELSE NULL
+            END AS validity
+        FROM
+            uploads u1
+            LEFT JOIN uploads_for_treasury_export ue ON ue.id = u1.id
+            LEFT JOIN users uv ON uv.id = u1.validated_by
+            LEFT JOIN users ui ON ui.id = u1.invalidated_by
+            JOIN reporting_periods rp ON rp.id = u1.reporting_period_id
+            JOIN agencies a ON a.id = u1.agency_id
+        WHERE
+            u1.tenant_id = ?
+        ORDER BY
+            rp.id,
+            ue.id,
+            u1.validated_at ASC
+    `, [organizationId, organizationId]);
+    return uploads.rows;
+}
+
+async function generateAndUploadMetadata(organizationId, s3Key, logger = log) {
+    const uploads = await module.exports.getUploadsForArchive(organizationId);
+    let data = `upload_id,original_filename,path_in_zip,agency_name,ec_code,reporting_period_name,validity`;
+    for (const upload of uploads) {
+        data = data.concat('\n', `${upload.upload_id},${upload.original_filename},${upload.path_in_zip},${upload.agency_name},${upload.ec_code},${upload.reporting_period_name},${upload.validity}`);
+    }
+    const fileExportParams = {
+        Bucket: process.env.AUDIT_REPORT_BUCKET,
+        Key: s3Key,
+        Body: Buffer.from(data),
+        ContentType: 'text/plain',
+        ServerSideEncryption: 'AES256',
+    };
+
+    const s3 = aws.getS3Client();
+    try {
+        await s3.send(new PutObjectCommand(fileExportParams));
+    } catch (err) {
+        logger.error(err, 'failed to upload full file export metadata to S3');
+        throw err;
+    }
+    logger.info('finished generating and uploading ARPA full file export metadata');
+}
+
+async function addMessageToQueue(organizationId, email, logger = log) {
+    const archiveKey = zipFileKey(organizationId);
+    const metadataKey = metadataFsName(organizationId);
+    logger.child({ archiveKey, metadataKey });
+
+    await module.exports.generateAndUploadMetadata(organizationId, metadataKey, logger);
+    const message = {
+        s3: {
+            bucket: process.env.AUDIT_REPORT_BUCKET,
+            zip_key: archiveKey,
+            metadata_key: metadataKey,
+        },
+        organization_id: organizationId,
+        user_email: email,
+    };
+
+    const sqs = aws.getSQSClient();
+    try {
+        await sqs.send(new SendMessageCommand({
+            QueueUrl: process.env.ARPA_FULL_FILE_EXPORT_SQS_QUEUE_URL,
+            MessageBody: JSON.stringify(message),
+        }));
+    } catch (err) {
+        logger.error({ err }, 'failed to add full file export message to SQS queue');
```

Comment:
Given that the other `logger.error()` call have been updated, I'm assuming this is a just a remnant of using Bunyan that got missed in the recent updates.

FYI The first argument to Pino's `logger.error()` doesn't need to be an `err`-keyed object like it does with Bunyan – you can just pass the error directly and it will be outputted with whatever "error key" is configured on the logger, which helps it play nicer with downstream log consumers (e.g. Datadog). 
```suggestion
        logger.error(err, 'failed to add full file export message to SQS queue');
```

Reply:
Done 👍